src/domain/change/ChangeLogic.ts

```diff
@@ -13,7 +13,13 @@ let log = new Log('ChangeLogic.ts')
 
 export default class ChangeLogic {
 
-    async create(change: Change, tx: PgTransaction): Promise<EntityResult<Change>> {
+    orm: Orm
+
+    constructor(orm:Orm){
```

Comment:
Mach die Abhängigkeit zu dem ORM nicht in den Constructor, auch wenn das semantisch auf jeden Fall die richtige Idee wäre. Der Hintergrund ist, dass die Abhängigkeiten so kreuz und quer werden können, dass es einen Deadlock zwischen den Konstruktoren geben kann. Ist zwar hier konkret nicht so, aber damit man nicht zwei Patterns hat, machen wir es lieber so, dass die Abhängigkeit ein Property der Klasse mit Ausrufezeichen wird: `orm!: Orm`. Das Ausrufezeichen, damit man es nicht im Constructor initialisieren muss und damit man innerhalb der Klasse dann nicht jedes Mal prüfen muss, ob der ORM gesetzt ist.